*java.security.cert.CertificateException* *CertificateException* This exception 

public class CertificateException
  extends    |java.security.GeneralSecurityException|

|java.security.cert.CertificateException_Description|
|java.security.cert.CertificateException_Fields|
|java.security.cert.CertificateException_Constructors|
|java.security.cert.CertificateException_Methods|

================================================================================

*java.security.cert.CertificateException_Constructors*
|java.security.cert.CertificateException()|Constructs a certificate exception w
|java.security.cert.CertificateException(String)|Constructs a certificate excep
|java.security.cert.CertificateException(String,Throwable)|Creates a Certificat
|java.security.cert.CertificateException(Throwable)|Creates a CertificateExcept

*java.security.cert.CertificateException_Description*

This exception indicates one of a variety of certificate problems. 


*java.security.cert.CertificateException()*

public CertificateException()

Constructs a certificate exception with no detail message. A detail message is 
a String that describes this particular exception. 


*java.security.cert.CertificateException(String)*

public CertificateException(java.lang.String msg)

Constructs a certificate exception with the given detail message. A detail 
message is a String that describes this particular exception. 

    msg - the detail message. 

*java.security.cert.CertificateException(String,Throwable)*

public CertificateException(
  java.lang.String message,
  java.lang.Throwable cause)

Creates a CertificateException with the specified detail message and cause. 

    message - the detail message (which is saved for later retrieval by the {@link 
       #getMessage()} method). 
    cause - the cause (which is saved for later retrieval by the {@link #getCause()} 
       method). (A null value is permitted, and indicates that the cause is 
       nonexistent or unknown.) 

*java.security.cert.CertificateException(Throwable)*

public CertificateException(java.lang.Throwable cause)

Creates a CertificateException with the specified cause and a detail message of 
(cause==null ? null : cause.toString()) (which typically contains the class and 
detail message of cause). 

    cause - the cause (which is saved for later retrieval by the {@link #getCause()} 
       method). (A null value is permitted, and indicates that the cause is 
       nonexistent or unknown.) 

